packages/amplify-function-plugin-interface/src/index.ts

```diff
@@ -137,6 +137,10 @@ export type FunctionParameters = {
   lambdaLayers: LambdaLayer[];
   environmentVariables?: Record<string, string>;
   secretDeltas?: SecretDeltas;
+  template?: string;
+  defaultRuntime?: string;
+  skipAdvancedSection?: boolean;
+  skipNextSteps?: boolean;
```

Comment:
these aren't really parameters of the function, rather they are controls for the walkthrough. Is there a way we can define this behavior declaratively. ie the advanced section is automatically skipped if no advanced parameters are passed. same with next steps

Reply:
Though the parameter is called `functionParams`, the values are actually used to control the CLI prompt values (or workflow). The existing calls to this workflow requires to run through the advanced section and next steps prompts by default. So imo it makes sense to introduce these two flags so that it can be safely switched off without impacting the existing modules.